Sources/Gen/Xoshiro.swift

```diff
@@ -0,0 +1,32 @@
+/// An implementation of xoshiro256**: http://xoshiro.di.unimi.it.
+public struct Xoshiro: RandomNumberGenerator {
+  private var state: (UInt64, UInt64, UInt64, UInt64)
+
+  public init() {
+    state = zip(
+      .int(in: .min ... .max),
+      .int(in: .min ... .max),
+      .int(in: .min ... .max),
+      .int(in: .min ... .max)
+    )
+    .run()
+  }
+
+  public init(seed: UInt64) {
+    self.state = (seed, 18_446_744, 073_709, 551_615)
+    for _ in 1...10 { _ = self.next() } // perturb
+  }
```

Comment:
Rather than seed from exact state of `(UInt64, UInt64, UInt64, UInt64)` this logic gives us some better results.

Also seeding from `(0, 0, 0, 0)` causes a generator to call `next()` forever.